core/appeal/service.go

```diff
@@ -1040,31 +1032,31 @@ func (s *Service) GrantAccessToProvider(ctx context.Context, a *domain.Appeal, o
 }
 
 func (s *Service) checkExtensionEligibility(a *domain.Appeal, p *domain.Provider, policy *domain.Policy, activeGrant *domain.Grant) error {
-	AllowActiveAccessExtensionIn := ""
+	allowActiveAccessExtensionIn := ""
 
 	// Default to use provider config if policy config is not set
 	if p.Config.Appeal != nil {
-		AllowActiveAccessExtensionIn = p.Config.Appeal.AllowActiveAccessExtensionIn
+		allowActiveAccessExtensionIn = p.Config.Appeal.AllowActiveAccessExtensionIn
 	}
 
 	// Use policy config if set
 	if policy != nil &&
 		policy.AppealConfig != nil &&
 		policy.AppealConfig.AllowActiveAccessExtensionIn != "" {
-		AllowActiveAccessExtensionIn = policy.AppealConfig.AllowActiveAccessExtensionIn
+		allowActiveAccessExtensionIn = policy.AppealConfig.AllowActiveAccessExtensionIn
 	}
 
-	if AllowActiveAccessExtensionIn == "" {
+	if allowActiveAccessExtensionIn == "" {
 		return ErrAppealFoundActiveGrant
 	}
 
-	extensionDurationRule, err := time.ParseDuration(AllowActiveAccessExtensionIn)
+	extensionDurationRule, err := time.ParseDuration(allowActiveAccessExtensionIn)
 	if err != nil {
-		return fmt.Errorf("%w: %v: %v", ErrAppealInvalidExtensionDuration, AllowActiveAccessExtensionIn, err)
+		return fmt.Errorf("%w: %q: %v", ErrAppealInvalidExtensionDuration, allowActiveAccessExtensionIn, err)
 	}
 
 	if !activeGrant.IsEligibleForExtension(extensionDurationRule) {
-		return ErrGrantNotEligibleForExtension
+		return fmt.Errorf("%w: extension is allowed %q before grant expiration", ErrGrantNotEligibleForExtension, allowActiveAccessExtensionIn)
```

Comment:
Would this value `allowActiveAccessExtensionIn` be in `hrs` or `days`?

Reply:
this is in the golang duration format so it'll be in hours